src/pages/experiments/[experimentId]/data-exploration/components/embedding/ClusterPopover.jsx

```diff
@@ -23,20 +25,34 @@ const ClusterPopover = (props) => {
 
   const content = getContent();
 
+  let style = {};
+  if (popoverPosition) {
+    style = { position: 'absolute', left: popoverPosition.current.x + 20, top: popoverPosition.current.y + 20 };
+  }
+
+  /* eslint-disable react/jsx-props-no-spreading */
   return (
-    <div style={{ position: 'absolute', left: popoverPosition.current.x + 20, top: popoverPosition.current.y + 20 }}>
-      <Popover title='Add cell set' content={content} visible />
+    <div style={style}>
+      <Popover title={message} content={content} {...restOfProps}>
+        {children}
+      </Popover>
     </div>
   );
+  /* eslint-enable react/jsx-props-no-spreading */
```

Comment:
put the eslint disable at the start of the file instead

Reply:
I wouldn't, because it only applies to the prop spreading stuff inside the return statement. My rationale is it's best practice to only disable linting for rules like this for the segment that actually needs to bypass it.

Reply:
you are right actually, I agree